packages/account/src/Components/self-exclusion/__tests__/self-exclusion-wrapper.spec.tsx

```diff
@@ -8,10 +8,18 @@ jest.mock('@deriv/shared/src/utils/screen/responsive', () => ({
     ...jest.requireActual('@deriv/shared/src/utils/screen/responsive'),
     isMobile: jest.fn(),
 }));
-jest.mock('../self-exclusion-article', () => () => <div>SelfExclusionArticle</div>);
+
+jest.mock('../self-exclusion-article', () => {
+    const mockArticle = () => <div>SelfExclusionArticle</div>;
```

Comment:
You can move this line (`const mockArticle = () => <div>SelfExclusionArticle</div>;`) out of the jest.mock function

Reply:
@niloofar-deriv If i move the variable out of the jest.mock, it will fail my test case stating that the element type is invalid 🙏 
![Screenshot 2023-01-16 at 12 02 54 PM](https://user-images.githubusercontent.com/103026762/212596159-f0c090ec-056a-4fd7-8bf4-35f310bf598f.png)